src/dotnet/UI.Blazor.App/Components/ChatList/chat-list.css

```diff
@@ -254,6 +254,9 @@ body.hoverable .chat-list .navbar-item:hover .navbar-item-ending .slot.hoverable
     @apply mx-0;
     @apply text-primary-title;
 }
+.navbar-group.place .tab-panel .tab-panel-tabs {
```

Comment:
This is not needed.
We should hide tabs only when search mode is on.
In chats it happens because we cover this area with search results.